HISTORY.md

```diff
@@ -6,6 +6,16 @@ The format is based on [Keep a
 Changelog](https://keepachangelog.com/en/1.0.0/), and this project
 adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [4.1.5](https://github.com/uploadcare/pyuploadcare/compare/v4.1.4...v4.1.5) - 2023-10-30
```

Comment:
Mix it into 4.2 update?

Reply:
Perhaps. Depends on when and in which order these two pull requests will be merged.